src/azure-cli/azure/cli/command_modules/vm/_validators.py

```diff
@@ -2129,6 +2129,16 @@ def _disk_encryption_set_format(cmd, namespace, name):
 # endregion
 
 
+def process_ppg_create_namespace(namespace):
+    """
+    The availability zone can be provided only when an intent is provided
+    """
+    if namespace.zone and not namespace.intentvmsizes:
+        raise RequiredArgumentMissingError('Please use parameter --intentvmsizes to specify possible sizes of virtual '
+                                           'machines that can be created in the proximity placement group.')
```

Comment:
> The availability zone can be provided only when an intent is provided

I suggest reflecting the relationship between `--intentvmsizes` and `--zone` in the error message